### PR TITLE
Port to Unreal 4.26

### PR DESCRIPTION
**Pull request by TaniaChistyakova**

### CODE REVIEW

File: Plugins/UXTools/UXTools.uplugin [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
 	"FileVersion": 3,
-	"Version": 1,
-	"VersionName": "1.0",
+	"Version": 2,
+	"VersionName": "0.9.0",
```
Comment (Cameron-Micka) [inline Plugins/UXTools/UXTools.uplugin:4]: Out of curiosity, did upgrading to UX Tools 0.10 have too many breaking changes?

File: KippysEscape.uproject [hunk 2]
```diff
@@ -34,6 +33,10 @@
 				"HoloLens",
 				"Win64"
 			]
+		},
+		{
+			"Name": "EditorScriptingUtilities",
```
Comment (Cameron-Micka) [inline KippysEscape.uproject:38]: Do you know what brought in this new dependency?

**Review by Cameron-Micka — APPROVED** [review]
Pulled down the changes and ran on HoloLens 2. Everything looked great to me. Thank you for upgrading this project! Added a few minor questions below.

**Review by sw5813 — APPROVED** [review]
Thanks for doing this Andrey!

File: Plugins/UXTools/UXTools.uplugin (hunk 1 above)
Comment (TaniaChistyakova) [reply Plugins/UXTools/UXTools.uplugin:4]: It was 0.9 early prerelease, but marked as 1.0 (that version was never released). I updated to the latest 0.9 to resolve all issues.

File: KippysEscape.uproject (hunk 2 above)
Comment (TaniaChistyakova) [reply KippysEscape.uproject:38]: Some scripts. EditorScriptingUtilities code initially were in the engine, but was moved to a plugin for 4.26.

**Pull request merged**